src/services/ubi/community.ts

```diff
@@ -1597,10 +1843,51 @@ export default class CommunityService {
             });
         }
 
-        if (fields.metrics) {
+        const stateExclude = ['id', 'communityId'];
+
+        const yesterdayDateOnly = new Date();
+        yesterdayDateOnly.setUTCHours(0, 0, 0, 0);
+        yesterdayDateOnly.setDate(yesterdayDateOnly.getDate() - 1);
+        const yesterdayDate = yesterdayDateOnly.toISOString().split('T')[0];
+
+        const dailyState = fields.dailyState
+        ? fields.dailyState.length > 0
+            ? fields.dailyState
+            : { exclude: stateExclude }
+        : [];
+        extendedInclude.push({
+            attributes: dailyState,
+            model: this.ubiCommunityDailyState,
+            as: 'dailyState',
+            duplicating: false,
+            where: {
+                date: yesterdayDate
+            },
+            required: false,
+        });
+
+        const stateAttributes = fields.state
+            ? fields.state.length > 0
+                ? fields.state.filter(
+                      (el: string) => !stateExclude.includes(el)
+                  )
+                : { exclude: stateExclude }
+            : [];
+        extendedInclude.push({
+            model: this.ubiCommunityState,
+            attributes: stateAttributes,
+            as: 'state',
+        });
```

Comment:
I know this is going to be tricky, but we must remove this table. Remove all it's instances.

Reply:
removed

Reply:
oh no, not removing removing. Removing the dependency on this table. We still need the state.